manifests/install.pp

```diff
@@ -28,14 +31,18 @@
   file { $teleport::extract_path:
     ensure => $directory_ensure,
   } ->
-  archive { $teleport::archive_path:
-    ensure        => $ensure,
-    extract       => true,
-    extract_path  => $teleport::extract_path,
-    source        => $teleport::archive_url,
-    creates       => "${teleport::extract_path}/teleport",
-    checksum      => $teleport::checksum,
-    checksum_type => $teleport::checksum_type,
+  exec { 'download-teleport-latest':
+    command => "/usr/bin/aws s3 --region=${::region} cp ${teleport::archive_url} ${teleport_latest_path}",
+  } ->
+  file { $teleport_current_path:
+    source => "file://${teleport_latest_path}",
+  } ~>
+  exec { 'extract-teleport-current':
+    refreshonly => true,
+    command => "/bin/tar xf ${teleport_current_path} -C ${teleport::extract_path} &&
+    chown root:root ${teleport::extract_path}/teleport/tctl &&
```

Comment:
👍for the `chown`. 

We should take some time to revisit how we run Teleport Node for any obvious permission fails, I've been meaning to do that but never got to it.